### PR TITLE
Create htTml-css form

### PR DESCRIPTION
**Pull request by ehs9nino**
created html form with CSS style

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# code-review
```
Comment (ehs9nino) [inline README.md:1]: For a project to understand it is better if you add the details of your project in the readme file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+  
+</head>
+<body>
+    <h1 id= "title">freeCodeCamp Survey Form</h1>
+    <p id ="description">Thank you for taking the time to help us improve the platform</p>
```
Comment (ehs9nino) [inline index.html:13]: It is better if you write code in a semantic way like you can use <section> tag to cover whole section.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,181 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap');
+
```
Comment (ehs9nino) [inline style.css:2]: I see no linter's configuration file, that will help you to write clean code and it will be good for understanding.

**Review by ehs9nino — COMMENTED** [review]
The forms look really okay but in order to write code professionally, you have to do the following steps.
1. Must add a readme file for a better understanding of your project.
2. Use the linters to write clean code.
3. Use the HTML semantics ( I pointed out them in a detailed comment )

After these changes, your code is written professionally. Happy Coding!

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @ehs9nino,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Required Changes ♻️

- [ ] Kindly phrase the comments politely, as suggestions instead of orders.
- For example:
   - This is a polite suggestion: "I think it would be better to use semantic tags"
   - This reads like an order: "Use semantic tags"

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ehs9nino) [reply index.html:13]: I think it's better if you use semantic tags in your HTML files i.e <section>

**Review by ehs9nino — COMMENTED** [review]
The forms look really okay but in order to write code professionally, I think it's better to do the following steps:
1. Adding a  readme file for a better understanding of your project will be really helpful.
2. I think adding the linters for this project is good in order to write clean code.
3. I think adding HTML semantics will be really a good point to write HTML documents perfectly  ( I pointed out them in a detailed comment )
After these changes, your code is written professionally. Happy Coding!

**Review by aimemalaika — APPROVED** [review]
Hi @ehs9nino,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**